sandbox/version-control-manager/pom.xml

```diff
@@ -8,8 +8,13 @@
     <artifactId>version-control-manager</artifactId>
     <version>0.0.1-SNAPSHOT</version>
     <dependencies>
+    	<dependency>
+		   	<groupId>com.sun.jersey</groupId>
+			<artifactId>jersey-client</artifactId>
+			<version>1.8</version>
+		</dependency>
```

Comment:
Please follow the formatting. Otherwise, the code looks messy. Use google-java-formatter